.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I can always convert it to JS 😄

the parser options is required to get some rules of `@typescript-eslint/parser` working.. else it would complain about the missing project :-(

Reply:
`.eslintrc` only is fine 🙆‍♂️ 

You shouldn't need to explicitly declare the `project` if your `tsconfig.json` doesn't exclude a file you wish to lint: https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex/rules/typescript.js#L15

Reply:
nice ✨